addons/launcher-addon/src/main/java/io/fabric8/launcher/addon/BoosterCatalogFactory.java

```diff
@@ -33,15 +32,15 @@
  */
 @ApplicationScoped
 public class BoosterCatalogFactory {
-    public static final String CATALOG_GIT_REPOSITORY_PROPERTY_NAME = "LAUNCHER_BACKEND_CATALOG_GIT_REPOSITORY";
+    private static final String LAUNCHER_BOOSTER_CATALOG_REPOSITORY = "LAUNCHER_BOOSTER_CATALOG_REPOSITORY";
```

Comment:
These system properties are set in https://github.com/fabric8-launcher/launcher-backend/blob/master/openshift/template.yaml and also in https://github.com/fabric8-launcher/launcher-openshift-templates/blob/master/openshift/launcher-template.yaml. 

Make sure to change that too.

Reply:
Good spot, thank you!

Reply:
Template updated and PR open https://github.com/fabric8-launcher/launcher-openshift-templates/pull/35